casts/splitNavigator/src/SplitNavigator/TabScreen.tsx

```diff
@@ -1,46 +1,67 @@
 import * as React from 'react';
-import { StyleSheet, ViewStyle, StyleProp } from 'react-native';
-import { Freeze } from 'react-freeze';
-import Animated, { useAnimatedStyle, useSharedValue, withSpring } from 'react-native-reanimated';
-
-import { ResourceSavingScene } from './ResourceSavingScene';
+import { StyleSheet, Animated as RNAnimated } from 'react-native';
+import { MaybeScreen } from './ScreenFallback';
 
 type TabScreenProps = {
     isVisible: boolean;
     children: React.ReactNode;
-    style?: StyleProp<ViewStyle>;
 };
-export function TabScreen({ isVisible, style, children }: TabScreenProps) {
-    const visible = useSharedValue(false);
-    const opacity = useSharedValue(0);
+
+function useLazyRef<T>(init: () => T): T {
+    const ref = React.useRef<T>(null);
+    if (ref.current == null) {
+        // @ts-ignore
+        ref.current = init();
+    }
+    return ref.current;
+}
+
+export function TabScreen({ isVisible, children }: TabScreenProps) {
+    /**
+     * Reanimated had a bug (possibly due to Freeze) on web
+     * when after stack push animation a screen become invisible
+     * (opacity was set to 0, even though a SharedValue wasn't 0).
+     */
```

Comment:
```suggestion
    /**
     * Reanimated had a bug (possibly due to Freeze) on web
     * when after a stack push animation a screen becomes invisible
     * (opacity was set to 0, even though a SharedValue wasn't 0).
     */
```

That's why you've removed `react-freeze`?

Reply:
I see, that why you've switched to Animated from RN :)

Reply:
I hadn't removed it actually, it's just `Screen` already has a `Freeze` container in it, so no need to do it twice